src/screens/mentee-home/mentee-home.tsx

```diff
@@ -42,11 +42,19 @@ const MenteeHome: React.FC = () => {
     []
   );
 
+  const reloadOpportunities = useCallback(() => {
+    dispatch(opportunityActions.loadOpportunities());
+  }, [dispatch]);
+
+  const reloadList = useCallback(() => {
+    reloadOpportunities();
+  }, [reloadOpportunities]);
+
```

Comment:
Maybe it's enough to have only reloadOpportunities function?

Reply:
There also will be a reloadNotifications function